src/js/admin/index.js

```diff
@@ -0,0 +1,25 @@
+document.querySelector( '.media-single-image button.browse' ).addEventListener( 'click', selectImage );
+
+function selectImage() {
+	const optionName = this.dataset.option;
+	let imageFrame = null;
+
+	if ( imageFrame ) {
```

Comment:
In what usecase would imageFrame be true-ish here?

Reply:
None 😄

Removed this.